### ISSUE TITLE
Unit Site Accessibility Review

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/uls#866]
I wanted to run through the unit sites that are live in SiteImprove to see if there were any issues that have come up. I've found some and will document the issues here that I'm fixing and what I'm reaching out to stakeholders to fix.

## Africana
Stakeholders:
1. On https://africana.library.cornell.edu/black-history-is-more-than-a-month/, HTML is used to style text. It looks like it’s Eric’s name and the paragraph below it. In the code, there’s align=”center” which is what’s triggering the error. If you edit the code and remove that, it’ll fix that.
2. Your most recent blog post, John Henrik Clarke and COVID-19, the image is missing alt text.
3. The “b” tag is being used to format text on https://africana.library.cornell.edu/the-john-henrik-clarke-africana-library-as-an-agent-of-change/ and https://africana.library.cornell.edu/the-sistahs/. These need to be changed to “strong” tags.
4. On https://africana.library.cornell.edu/covid-19/, the linked text in the body that says “John Henrik Clarke” is the same text the tag uses to link in the footer and they go to two different locations. I’d recommend changing the link text in the body of the article.
5. On https://africana.library.cornell.edu/the-john-henrik-clarke-africana-library-as-an-agent-of-change/, the linked text in the body that says “John Henrik Clarke Africana Library” is the same text the Library name in the header uses. I’d recommend changing the link text in the blog post to “mission of the John Henrik Clark Africana Library” to fix this.
6. On https://africana.library.cornell.edu/the-sistahs/, the text for links for Angela Davis and Women’s History Month are used twice, both going to different locations and one of each will need changed.
7. On https://africana.library.cornell.edu/black-history-is-more-than-a-month/, “Willard Straight Occupation” is linked in the body text to a different location than it is in the navigation menu. The one in the blog post would need changed.

Kevin:
1. Updated Thesis single template, was missing the ID of main-content for the skip link to be valid.
2. Updated a few links with duplicate locations that were related to navigation.

## Makerspace
No issues

## Engineering
Stakeholders:
1. On the homepage, 3 links are only identified by color. Under Research Starting Points, they are Engineering Research, How-to library tutorials, and Reserve a Classroom. 
2. On https://engineering.library.cornell.edu/resources/ and https://engineering.library.cornell.edu/sticker-shock/, it looks like the mailto engrref@cornell.edu link doesn’t match what’s in the footer.

Kevin:
1. Updated navigation link to match what is used in homepage text for Passkey.

**Comment by kevinkidwell** [comment]
## Johnson
Stakeholders:
1. On https://johnson.library.cornell.edu/use-the-library/, the Workshops link doesn’t match what is in the main navigation and footer. This link should be updated to match those or have the link text changed.
2. On https://johnson.library.cornell.edu/faqs/what-are-the-rules-for-borrowing-course-reserve-material-from-the-management-library/, the Course Reserves link doesn’t go to the same location as the navigation and should be updated to match.
3. On https://johnson.library.cornell.edu/alumni-and-friends/, the two links for Research Hours go to different locations and should either have the same link or different link text.
4. In Databases, the Ad$pender logo is missing alt text.
5. On the homepage, the Feature Resource image of the purple chat bubble with a question mark is missing alternative text. Since it’s a link, it should describe where the link goes.

Kevin:
1. Course Reserves module is missing a submit button.

**Comment by kevinkidwell** [comment]
## Physical Sciences
Stakeholders:
No issues.

Kevin:
1. Navigation had an incorrect link, made it match page content.
2. Color contrast is insufficient on the covid banner, ran a re-check of all pages to clear.

**Comment by kevinkidwell** [comment]
## Mann
Stakeholders:
1. Color contrast in navbar for Library Catalog, Course Reserves, and Ask a Librarian link on hover.
2. On https://mannlib.cornell.edu/news-events/news/the-library-is-virtually-open, "u" tag used.

## Asia
Kevin (these issues will be easier/faster for me to fix than to contact stakeholders):
1. Multiple pages with link text that is used for different destinations.
2. Images missing alt text on 2 pages.

**Comment by JesiBuell79** [comment]
Will be running accessibility checks differently so closing this ticket
